user/yaml/README.md

```diff
@@ -0,0 +1,327 @@
+```
+
+Then follow the same instructions for the Kind cluster deployment.
+
+## Access **KubeStellar** service directly from the host OS without `admin.kubeconfig` or plugins
+
+Since **kubectl**, **kcp** plugins, and **KubeStellar** executables are include in the **KubeStellar** container image we can operate KubeStellar directly from the host OS using `kubectl`, for example:
+
+```shell
+$ kubectl exec -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}) -- kubectl ws tree
+.
+└── root
+    ├── compute
+    └── espw
+
+$ kubectl exec -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}) -- kubectl ws create imw
+Workspace "imw" (type root:organization) created. Waiting for it to be ready...
+Workspace "imw" (type root:organization) is ready to use.
+```
+
+## Access **KubeStellar** service from the host OS by extracting the `admin.kubeconfig` and the plugins from the pod
+
+In this case the host OS will need a copy of **kcp** plugins, and **KubeStellar** plugins, assuming that the architecture of the client is the same of the container, the executables can be extracted from the container:
+
+```shell
+kubectl exec -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}) -- tar cf - "/home/kubestellar/kcp-plugins" | tar xf - --strip-components=2
+
+kubectl exec -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}) -- tar cf - "/home/kubestellar/kubestellar" | tar xf - --strip-components=2
+```
+
+As an alternative the bootstrap script can be used with the `--deploy false` option as follows:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/main/bootstrap/bootstrap-kubestellar.sh) --deploy false
+```
+
+The `admin.kubeconfig` to access **KubeStellar** from outside the cluster can be obtained by:
+
+```shell
+kubectl cp -n kubestellar $(kubectl get pod -n kubestellar --selector=app=kubestellar-server -o jsonpath={.items[0].metadata.name}):/home/kubestellar/.kcp/external.kubeconfig ./admin.kubeconfig
+```
+
+or
+
+```shell
+kubectl get secrets kubestellar -n kubestellar -o 'go-template={{index .data "external.kubeconfig"}}' | base64 --decode > ./admin.kubeconfig
+```
+
+Then, the following environment variables need to be set:
+
+```shell
+export KUBECONFIG=$PWD/admin.kubeconfig
+export PATH=$PATH:$PWD/kcp-plugins/bin:$PWD/kubestellar/bin
+```
+
+If necessary, add the **KubeStellar** ingress, *e.g.* `kubestellar.svc.cluster.local`, to the `/etc/hosts` files:
+
+```text
+127.0.0.1       kubestellar.svc.cluster.local
+```
+
+Now we can use use **KubeStellar** in the usual way:
+
+```shell
+$ kubectl ws tree
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+## Access **KubeStellar** from another pod in the same `kubestellar` namespace

```

Comment:
No harm in writing this down for us KubeStellar contributors, but I think this use case is of no interest to users.

Reply:
This is @dumb0002 use case for MCAD

Reply:
He plans to put MCAD in the KubeStellar namespace?  That would be unusual.  Normally an independent project like MCAD would put its Kubenetes API objects in its own Kubernetes namespace.

Reply:
ok, probably not the same namespace specifically, I will reword the title accordingly